src/footprints/via.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    nets: ['from'],
+    body: p => `
+
+	(module VIA-0.6mm (layer F.Cu) (tedit 591DBFB0)
+	
+        ${p.at /* parametric position */}   
+		  (fp_text reference REF** (at 0 1.4) (layer F.SilkS) hide

```

Comment:
References and values should look something like:

```
(fp_text reference "${p.ref}" (at 0 -8.255) (layer F.SilkS) ${p.ref_hide}
    (effects (font (size 1 1) (thickness 0.15))))
(fp_text value "" (at 0 8.25) (layer F.Fab)
    (effects (font (size 1 1) (thickness 0.15))))
````

So, both `ref` and `ref_hide` come from the function params, and value is empty for consistency.